.github/workflows/test.yml

```diff
@@ -24,5 +24,5 @@ jobs:
       - name: Test
         env:
           MNEMONICS: ${{ secrets.MNEMONICS }}
-          NETWORK: ${{ secrets.NETWORK }}
+          NETWORK: dev
```

Comment:
same as for integration.yml, we need an issue to track this

Reply:
- https://github.com/threefoldtech/grid3-go/issues/113